cinema/views.py

```diff
@@ -0,0 +1,47 @@
+from django.shortcuts import render, get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def movie_info(request, pk):
+    movie = get_object_or_404(Movie, pk=pk)
+
```

Comment:
read this documentation it will help to make your code better https://www.django-rest-framework.org/tutorial/1-serialization/
and add an annotation type

Reply:
I would like some context regarding those improvements...

Reply:
This documentation is on serialization we need the requests-and-responses one

Reply:
annotation type:
We were told that its is not required

Reply:
<img width="850" alt="image" src="https://user-images.githubusercontent.com/108864360/202160640-fd960053-4c20-4aa5-8856-765d6085d564.png">

Reply:
get_object_or_404 is the same as the method listed in docs so i disagree

Reply:
> get_object_or_404 is the same as the method listed in docs so i disagree

Good